[Bug]: Compilation fails on master due to undeclared AtomicU32 in src/channels/mod.rs

### Affected component

unknown

### Severity

S1 - workflow blocked

### Current behavior

After PR #3432 
A fresh clone of the `master` branch fails to compile on 32-bit targets. The file `src/channels/mod.rs` attempts to initialize an `AtomicU32` counter (`AtomicU32::new(1)`) around line 2355 without actually importing `AtomicU32` into scope, resulting in an `E0433` compiler error. 

Additionally, there is a leftover/unused import for `portable_atomic::{AtomicU64, Ordering}` around line 89. This appears to be a conflicting merge artifact from recently merged PRs. 

### Expected behavior

The `master` branch should compile successfully out of the box. `src/channels/mod.rs` should either properly import `std::sync::atomic::AtomicU32` for 32-bit targets or use the fully qualified path (`std::sync::atomic::AtomicU32::new(1)`). The unused `portable_atomic` import should also be cleaned up.

### Steps to reproduce

```bash
1. Clone the latest `master` branch.
2. Run a build targeting a 32-bit architecture (e.g., `cargo +nightly build --release --target mipsel-unknown-linux-musl`).
3. Observe the `E0433` compiler crash in `src/channels/mod.rs`.
```

### Impact

* **Affected users:** Anyone compiling the `master` branch for 32-bit edge devices or using `target_has_atomic` fallbacks. 
* **Frequency:** Always. 
* **Consequence:** Hard block on compilation until the source file is manually patched. 

### Logs / stack traces

```text
error[E0433]: cannot find type `AtomicU32` in this scope
    --> src/channels/mod.rs:2355:34
     |
2355 |     let task_sequence = Arc::new(AtomicU32::new(1));
     |                                  ^^^^^^^^^ use of undeclared type `AtomicU32`

warning: unused import: `AtomicU64`
  --> src/channels/mod.rs:89:23
   |
89 | use portable_atomic::{AtomicU64, Ordering};
   |
```

### ZeroClaw version

master (latest commit)

### Rust version

nightly

### Operating system

mipsel-unknown-linux-musl

### Regression?

Unknown

### Pre-flight checks

- [x] I reproduced this on the latest master branch or latest release.
- [x] I redacted secrets, tokens, and personal data from all submitted content.

--- a/src/channels/mod.rs
+++ b/src/channels/mod.rs
@@ -2407,10 +2407,7 @@ async fn run_message_dispatch_loop(
         String,
         InFlightSenderTaskState,
     >::new()));
-    #[cfg(target_has_atomic = "64")]
     let task_sequence = Arc::new(AtomicU64::new(1));
-    #[cfg(not(target_has_atomic = "64"))]
-    let task_sequence = Arc::new(AtomicU32::new(1));
 
     while let Some(msg) = rx.recv().await {
         let permit = match Arc::clone(&semaphore).acquire_owned().await {

